paperspace/cli/experiments.py

```diff
@@ -108,11 +108,11 @@ def common_experiments_create_options(f):
 def common_experiment_create_multi_node_options(f):
     options = [
         click.option(
-            "--experimentTypeId",
-            "experimentTypeId",
+            "--experimentType",
```

Comment:
actually it would be enough to just just change this line :) second param is name of internal parameter, first is for the user.